cli/daemon/dash/dashapp/src/components/api/api.ts

```diff
@@ -1,67 +1,5 @@
-import {Decl, Loc} from "./schema";
+import * as pb from '../../../../../../../proto/encore/parser/meta/v1/meta.pb'
+export * from '../../../../../../../proto/encore/parser/meta/v1/meta.pb'
```

Comment:
Honestly, I tried to do that, but I couldn't get it to work - I spent about 15-20 minutes on it because deciding on these two files it's not that bad.